nuxeo-services/login/nuxeo-platform-login-saml2/src/main/java/org/nuxeo/ecm/platform/auth/saml/SAMLAuthenticationProvider.java

```diff
@@ -254,6 +258,27 @@ private void addProfile(AbstractSAMLProfile profile) {
         profiles.put(profile.getProfileIdentifier(), profile);
     }
 
+    private static final String SIGNATURE_ALGORITHM = "SignatureAlgorithm";
+
+    private static final String DIGEST_ALGORITHM = "DigestAlgorithm";
+
+    private void initializeSecurity(Map<String, String> parameters) {
+        BasicSecurityConfiguration config = (BasicSecurityConfiguration) Configuration.getGlobalSecurityConfiguration();
+        for (Entry<String, String> es : parameters.entrySet()) {
+            String key = es.getKey();
+            String value = es.getValue();
+            if (key.startsWith(SIGNATURE_ALGORITHM)) { //
```

Comment:
```suggestion
            if (key.startsWith(SIGNATURE_ALGORITHM)) {
```
or missing comment maybe?

Reply:
Unfinished comment but that's not really needed the code is clear enough now.